src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -293,15 +293,12 @@ const NavBar: React.FC<NavBarPropsType> = ({
                   overlay={ProfilePopover}
                   rootClose={true}
                 >
-                  <div className="navbar-link-profile">
-                    <img
-                      src={profileImage}
-                      className="profile-icon"
-                      alt="avatar"
-                      width="40"
-                      height="40"
-                    />
-                  </div>
+                  <UserAvatar
+                    user={profile}
+                    containerClassName="navbar-profile-image-container"
+                    statusClassName="navbar-profile-image-status"
```

Comment:
I guess it just has different size? In this case we should define the `size` property in UserAvatar and handle it inside

Reply:
What's the different size? I almost added a `size` prop to `UserAvatar` in a previous PR, but then didn't bother. I'd be open to us adding it.

The way I implemented it was with a `AvatarSize` `enum`, with values like `default`, `small`, etc as needed. I then made 'size --variants' in the scss, and applies these with `classNames` based on the value of the `size` prop enum values.

That way we can consistently configure the couple of different avatar sizes we should need across the platform, which IMO will make it easier to keep consistent rather than allowing any arbitrary size params to be passed in.

Reply:
@0xdevalias prop 'large' added. It is enough for this component

Reply:
@oleksii-front Can you please add the commit hash to these comment threads of where you make the changes to make it easy for us to review.

Reply:
https://github.com/sparkletown/sparkle/pull/1384/commits/76498f6cf6998b09e94d345c54d36a4a9d199f96